ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/claimantresponse/ResponseAcceptationMapper.java

```diff
@@ -25,7 +25,7 @@ public ResponseAcceptationMapper(PaymentIntentionMapper paymentIntentionMapper,
     @Override
     public CCDResponseAcceptation to(ResponseAcceptation responseAcceptation) {
         CCDResponseAcceptation.CCDResponseAcceptationBuilder builder = CCDResponseAcceptation.builder();
-        builder.formaliseOption(CCDFormaliseOption.valueOf(responseAcceptation.getFormaliseOption().name()));
+        builder.formaliseOption(CCDFormaliseOption.valueOf(responseAcceptation.getFormaliseOption().get().name()));
```

Comment:
`Optional.get()` without checking its presence. Replace your `Optional.get()` calls with `orElse` or `orElseThrow` baed on the scenario.